contentcuration/contentcuration/frontend/channelEdit/components/edit/LevelsOptions.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+  <VSelect
+    ref="level"
+    v-model="level"
+    :items="levels"
+    box
+    chips
+    clearable
+    :label="translateMetadataString('level')"
+    multiple
+    deletableChips
+  />
+
+</template>
+
+<script>
+
+  import { camelCase } from 'lodash';
+  import { ContentLevels } from 'shared/constants';
+  import { constantsTranslationMixin, metadataTranslationMixin } from 'shared/mixins';
+
+  export default {
+    name: 'LevelsOptions',
+    mixins: [constantsTranslationMixin, metadataTranslationMixin],
+    props: {
+      value: {
+        type: Array,
+        default: () => [],
+      },
+    },
+    computed: {
+      level: {
+        get() {
+          return this.value;
+        },
+        set(value) {
+          this.$emit('input', value);
+        },
+      },
+      /**
+       * List of levels to show in the dropdown, taking into account mapping
+       * to levels in the constants that do not have an exact matching corresponding string
+       * @returns {Array}
+       */
+      levels() {
+        return Object.entries(ContentLevels).map(level => {
+          let translationKey;
+          if (level[0] === 'PROFESSIONAL') {
+            translationKey = 'specializedProfessionalTraining';
```

Comment:
cc @marcellamaki we might want to consolidate this logic somewhere like I think we did on Kolibri?

Reply:
I think I stole that from Kolibri: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/SelectGroup.vue#L139